src/material-experimental/mdc-autocomplete/autocomplete.ts

```diff
@@ -35,7 +36,8 @@ import {
   },
   providers: [
     {provide: MAT_OPTION_PARENT_COMPONENT, useExisting: MatAutocomplete}
-  ]
+  ],
+  animations: [panelAnimation],
```

Comment:
Should we have this on the non-MDC version too? I think that it may make the switch more difficult, because the timing in people's tests will change.

Reply:
My guess is adding it in the current one would be too hard to get in. We can try it at least in a separate PR